.github/workflows/conformance-aws-cni.yaml

```diff
@@ -66,7 +66,7 @@ concurrency:
 env:
   clusterName: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
   region: us-east-2
-  cilium_cli_version: v0.10.4
+  cilium_cli_version: v0.11.1
```

Comment:
Cannot comment on the actual line, but it looks like this workflow (and the ones for v1.10 and v1.11) use a mix of `helm install` and `cilium` CLI. Should we switch this to use `cilium` CLI consistently, like in other workflows?

Reply:
That is correct, they use `helm install` but only use `cilium` CLI for the connectivity tests. I agree that we should change cilium cli for conformance aws but should be done a as a follow up PR.

Reply:
FYI, I don't have any TODO on my side to fix this. As Nicolas said, it would require the CLI to support chaining mode installations first.